backend/main.py

```diff
@@ -52,8 +54,8 @@
     return dict
 
 
-@app.post("/write_events")
-async def write_event(event):
+@app.route("/write_events", methods=['POST'])
+def write_event(event):
     # Change the scope to 'https://www.googleapis.com/auth/calendar' and delete any
```

Comment:
This is my fault not yours, but could you please add comments that explain the purpose of the function (similar to the read_events())